### DISCUSSION TITLE
GUI for your art engine

### DISCUSSION THREAD

**Discussion by robin-liquidium** [nftchef/art-engine#142] [Ideas]
Hey nftchef, thanks for your great works so far. 
Do you think it's possible to work your features into the new art_engine_app (with GUI) by Hashlips so people who can't code can do more with it? 
Really appreciate you helping the creators!

**Comment by nftchef** [comment]
HI, it's totally possible, but I do not have any plans to do it myself in the near future. there simply isn't enough demand or time 🙇
